### PR TITLE
Allow PHP 8

### PR DESCRIPTION
**Pull request by endroid**
Hi just allowing PHP 8 works for my project. Maybe someone still has issues, we can resolve them here.

### CODE REVIEW

**Comment by prgrssm** [comment]
@endroid m4tthumphrey/php-gitlab-api should be not lower than 10.3.0. In this version they added PHP 8 support

**Comment by Zeichen32** [comment]
We also should add php 8 to the Travis build.
https://github.com/Zeichen32/GitLabApiBundle/blob/master/.travis.yml

**Comment by endroid** [comment]
Yes and we could safely drop support for 7.2 (EOL) I think?

**Comment by Zeichen32** [comment]
> Yes and we could safely drop support for 7.2 (EOL) I think?

I think we should follow the min versions of the GitlabClient and Symfony. Both currently requires minimal PHP 7.2.5.

**Comment by endroid** [comment]
@Zeichen32 You see anything that needs to be changed? Travis looks green now :)

File: composer.json [hunk 1]
```diff
@@ -12,17 +12,17 @@
...
         "symfony/http-client-contracts": "^1.0|^2.0",
         "nyholm/psr7": "^1.0",
-        "m4tthumphrey/php-gitlab-api": "^10.0"
+        "m4tthumphrey/php-gitlab-api": "^10.3"
     },
     "require-dev" : {
         "phpunit/phpunit": "^8.0",
         "guzzlehttp/psr7": "^1.5.2",
-        "php-http/guzzle6-adapter": "^2.0.1",
+        "php-http/guzzle7-adapter": "^0.1.1",
```
Comment (GrahamCampbell) [inline composer.json:25]: This line and the one above should be deleted, and replaced with `guzzlehttp/guzzle` at `^7.2`.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: composer.json [hunk 2]
```diff
@@ -22,8 +22,7 @@
     "require-dev" : {
         "phpunit/phpunit": "^8.0",
         "guzzlehttp/psr7": "^1.5.2",
-        "php-http/guzzle7-adapter": "^0.1.1",
-        "http-interop/http-factory-guzzle": "^1.0"
+        "guzzlehttp/guzzle": "^7.2"
```
Comment (GrahamCampbell) [inline composer.json:25]: No, you need http-interop/http-factory-guzzle, but not guzzlehttp/psr7.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (GrahamCampbell) [reply composer.json]: You replaced this line and the one below, instead of the one above. :P

File: composer.json (hunk 2 above)
Comment (endroid) [reply composer.json:25]: Ah :) I will fix that

**Comment by Zeichen32** [comment]
@endroid @GrahamCampbell  Any more comments on this PR or can i merge this PR?

**Review by GrahamCampbell — APPROVED** [review]

**Pull request merged**